releasenotes/notes/fix-double-colon-parsing-in-sql-obfuscator-fcf487472126da13.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    APM: fix parsing the double-colon operator in the SQL obfuscator.
```

Looking at this I have no idea what was fixed and what the problem was. Maybe
```suggestion
    APM: Allow double-colon operator in SQL obfuscator.
```
?